melos.yaml

```
@@ -22,3 +22,6 @@ scripts:
   format-check: dart format --output=none --set-exit-if-changed --line-length 120 .
   analyze: dart analyze --fatal-infos .
   test: melos exec --concurrency=1 --fail-fast --dir-exists=test -- "flutter test"
+  generate_neon: melos exec --scope="neon*" -- dart run build_runner build --delete-conflicting-outputs
+  format: melos exec dart format --fix --line-length 120 .
+  gen-l10n: melos exec --flutter --dir-exists="lib/l10n" flutter gen-l10n && melos run format
```

```suggestion
  generate:build_runner: melos exec --scope="neon*" -- dart run build_runner build --delete-conflicting-outputs
  format: melos exec dart format --fix --line-length 120 .
  generate:l10n: melos exec --flutter --dir-exists="lib/l10n" flutter gen-l10n && melos run format
```

What do you think about this?
I think it is better because the names use the same format.
I also changed it from `neon` to `build_runner` because that's more accurate and avoids confusion with our `generate-neon.sh` script.

---

I limited the scope to only run build_runner in `packages/neon` to avoid running dynamite/generating nextcloud wich can be a lot sometimes (and suspected to grow once we add more specs and fix oom).

I also named the l10n command to be the same as the flutter built in one but that doesn't need to be the case

---

Right, but then we still have confusion with the generate-neon.sh script?

---

what about `generate:neon:buildrunner` ?

---

Perfect